documentation/smg/04_building_with_cmake.md

```diff
@@ -1,38 +1,151 @@
 # Building with CMake
 
+## Build dependencies
+
+A developer building Herbert/Horace will need to install:
+
+- [Matlab](https://www.mathworks.com/products/matlab.html) >= 2018b
+  - on Linux the desired Matlab version must be on the path when running the
+  script
+  - on Windows, by default, the latest version is found via the registry
```

Comment:
If you look at smg/02 (new starter docs) I think you will see I described getting Matlab with STFC license through the STFC website - currently 2018 or 2019, not 2020. Guessing this would be better for us. Is this instructions for external users though? In that case, it would be useful to precede this section with a "Who this is for" section, which warns developers/internal users to look at smg/02 instead. Is there a class of internal users who aren't developers who would fall  through this crack?